source/stylesheets/base/_typography.scss

```diff
@@ -39,26 +39,43 @@ a {
   }
 }
 
-.link-light {
-  color: $blue;
-  background: url('../../images/link-background-light.svg') no-repeat center left;
-  background-size: contain;
-  padding: 3px 20px;
-}
-
-.link-medium {
-  color: $blue;
-  background: url('../../images/link-background-medium.svg') no-repeat center center;
+@mixin gen-link($color, $imgURL, $position, $imgURLHover) {
+  color: $color;
+  background: url($imgURL) no-repeat $position;
   background-size: contain;
   padding: 3px 20px;
   &:hover {
     a {
       color: $white;
     }
-    background: url('../../images/link-background-medium-hover.svg') no-repeat center center;
+    background: url($imgURLHover) no-repeat center center;
   }
 }
 
+.link-light {
+  @include gen-link(
+    $blue,
+    '../../images/link-background-light.svg',
+    center left,
+    '../../images/link-background-medium-hover.svg');
+}
+
+.link-medium {
+  @include gen-link(
+    $blue,
+    '../../images/link-background-medium.svg',
+    center center,
```

Comment:
You should be able to get away with one `center` here.

![screen shot 2017-10-17 at 4 08 10 pm](https://user-images.githubusercontent.com/5566826/31686836-6d446eca-b355-11e7-9de0-745dc757aa88.png)